### ISSUE TITLE
Always get memory warning because of the plugin

### ISSUE THREAD

**Issue by seyvillas-morteza** [nette-intellij/intellij-latte#205]
When I activate Latte or even Latte Pro I constantly get errors regarding the memory.

**Environment (please complete the following information):**
 - PhpStorm 2023.1, Build #PS-231.8109.199, built on March 29, 2023
 - Plugin version 1.1.7
 - MacBook Pro

**To Reproduce**
Steps to reproduce the behavior (or attach video):
1. Just install and use it

**Code**
None

**Screenshots**
<img width="816" alt="Screenshot 2023-04-13 at 10 40 11" src="https://user-images.githubusercontent.com/72194309/231707245-81d37d75-0672-4fb6-8b08-e8680b46104e.png">

**Additional context**
It seems that it has had this issue for so long!
I also noticed when the page gets bigger or more complicated or multiple include based on situations it also cannot reformat the page properly. Adds empty spaces for no reason.

**Comment by mesour** [comment]
@seyvillas-morteza yes, this plugin is not optimised for performance. I didn't know how to work with the IntelliJ Plugin SDK when I updated this plugin. When I learned this, I programmed a paid plugin: [Latte Pro](https://plugins.jetbrains.com/plugin/19661-latte-pro). Because it would be very difficult to optimize this plugin.
